### DISCUSSION TITLE
MOD3.0 Design issue #3: What class (outside of DCAT) can we consider a Semantic Artefact specialize?

### DISCUSSION THREAD

**Discussion by jonquet** [sifrproject/MOD-Ontology#36] [General]
- Properties of all the classes becomes “available” to describe SemanticArtefact
- We need to be sure the logics stays right

### Proposition 1: 
- SA specializes all the pre-existing semantic objects (e.g., owl:ontology) as it was done in MOD1.4 with the object mod:Ontology.
=> All the properties of all the objects are available.

<img width="782" alt="Capture d’écran 2023-04-10 à 15 07 30" src="https://user-images.githubusercontent.com/7379183/231008263-bb51c084-fe37-4346-977e-e72266e70e20.png">

### Proposition 2: 
- SA specializes even more pre-existing semantic objects but not the pre-existing ones that were representing a type of SA (owl:Ontology, skos:ConceptScheme, voaf:Vocabulary)
- 
<img width="499" alt="image" src="https://user-images.githubusercontent.com/7379183/231008436-ccab1287-c606-4c78-b6c5-337337cf0431.png">

### Proposition 2-bis: 
- The same than 2 but SA supersedes the pre-existing ones.
<img width="566" alt="Capture d’écran 2023-04-10 à 15 11 10" src="https://user-images.githubusercontent.com/7379183/231008665-677873d7-8220-4080-959a-83936b648a25.png">

**Comment by jonquet** [comment]
Properties available from other standard vocabularies

<img width="238" alt="image" src="https://user-images.githubusercontent.com/7379183/231008892-70ebe950-30e8-4e47-8751-49c33ffa9c37.png">

<img width="260" alt="image" src="https://user-images.githubusercontent.com/7379183/231008913-efcdc51e-c870-4955-9688-5df3182bfb10.png">

<img width="232" alt="image" src="https://user-images.githubusercontent.com/7379183/231008924-0931808b-b583-441c-b0f1-ce5fc4a959d8.png">

<img width="319" alt="image" src="https://user-images.githubusercontent.com/7379183/231008944-52ab0d7f-120e-402d-a980-8ec062d6c461.png">

<img width="254" alt="image" src="https://user-images.githubusercontent.com/7379183/231008962-3d53f489-760f-4683-a458-2c0463172c8a.png">

<img width="341" alt="image" src="https://user-images.githubusercontent.com/7379183/231008980-faa520a3-98c1-4524-b5a5-eb795ada1be4.png">

<img width="172" alt="image" src="https://user-images.githubusercontent.com/7379183/231009000-6c601d6a-eba5-4787-87d6-389606ef8a46.png">

<img width="333" alt="image" src="https://user-images.githubusercontent.com/7379183/231009019-89d9cf58-0e43-435a-97ae-62afdb0bc0cd.png">

**Reply by agbeltran** [reply]
The conclusion around this design issue considered:
mod:SemanticArtefact must not specialise owl:Ontology or rdfs:Resource
Any other specialisation, when considering other vocabularies, should be discussed as part of an MOD profile and/or MOD mappings with other vocabularies, rather than being part of MOD definitions.
